### ISSUE TITLE
Pylint confuses pyxb.utils with top-level utils package.

### ISSUE THREAD

**Issue by ut-adamc** [PyCQA/pylint#920]
Tested in python  3.4, so I will use that concrete version in the paths below.

### Steps to reproduce
1. Create a folder, pylint_bug, with an associated virtual environment (tested using pyvenv-3.4). Activate the virtual environment.
2. Install pyxb
3. Go to <virtualenv>/lib/python3.4/site-packages and create a folder utils. Give it the following structure:
```
utils/
    __init__.py (empty)
    bar/
        __init__.py (empty)
        foo.py (contains dummy class Foo)
```
4. Inside pylint_bug/, create test.py with the content
```python
from utils.bar.foo import Foo

print('ran ok')
```
### Current behavior
Run epylint on test.py results in:
1. test.py:1: error (E0611, no-name-in-module, ) No name 'bar' in module 'utils'
2. test.py:1: error (E0401, import-error, ) Unable to import 'utils.bar.foo'
3. test.py:1: warning (W0611, unused-import, ) Unused Foo imported from utils.bar.foo

Only the third item (W0611) is correct. Note that the program executes fine.

I am relatively certain that pylint is analyzing pyxb.utils instead of the correct top-level utils. If we change line 1 of test.py to "from utils import six", which makes sense for pyxb.utils but not utils per se, epylint will emit only the warning "test.py:1: warning (W0611, unused-import, ) Unused six imported from utils", although the program will fail with an import error in this configuration.

### Expected behavior
Getting the warning W0611, unused-import and nothing else.

### pylint --version output
pylint 1.5.5, 
astroid 1.4.5
Python 3.4.2 (default, Oct  8 2014, 10:45:20) 
[GCC 4.9.1]

**Comment by PCManticore** [comment]
Hi,

Thank you for reporting this issue and sorry for taking so long to land a fix.

The issue is not related to pyxb in anyway, but more likely to pylint itself. As described in commit message, the utils module of pylint was "imported" for analysis instead of the global utils module, which was a side effect of how epylint was calling pylint itself.

Please let me know if the solution works for you as well.

**Comment by ut-adamc** [comment]
This does work in my testing. Thanks!
